pkg/restapi/oidc/operations.go

```diff
@@ -141,6 +143,13 @@ func New(config *Config) (*Operation, error) {
 		return nil, fmt.Errorf("failed to open tokens store: %w", err)
 	}
 
+	if config.UserSDSURL != "" {
+		op.userSDSClient = client.New(
+			config.UserSDSURL+"/encrypted-data-vaults",
```

Comment:
Assume that the URL passed in has the `/encrypted-data-vaults` part in it. While our TrustBloc implementation uses that, other ones may not.

Reply:
(it'll also make it consistent with other projects - we include the `/encrypted-data-vaults` part in the URL passed in via the env. variables)

Reply:
[done](https://github.com/trustbloc/edge-agent/compare/f72677242d790bd9c9ea7e6785e9df7517f0ac84..e93ac5731b540cd1c7af5454293fd44f23408e1e)